app/src/main/java/com/mealsmadeeasy/ui/home/glance/WeekAtAGlanceMealViewHolder.kt

```diff
@@ -58,4 +64,40 @@ class WeekAtAGlanceMealViewHolder(
         })
         popup.show()
     }
+
+    private fun onClickEditServings() {
+        val root = View.inflate(itemView.context, R.layout.view_edit_servings, null)
+        val servingsText = root.findViewById<TextView>(R.id.add_to_plan_servings)
+        val plusButton = root.findViewById<ImageView>(R.id.add_to_plan_plus)
+        val minusButton = root.findViewById<ImageView>(R.id.add_to_plan_minus)
+
+        var numServings = mealPortion.servings
```

Comment:
When changing the number of servings and returning to this dialog, it resets to the original value. Is there an issue here or with the meal store?

Reply:
Just kidding. I can't reproduce this issue. 🤷‍♀️